src/main/java/org/broadinstitute/hellbender/utils/variant/writers/ReblockingGVCFBlockCombiner.java

```diff
@@ -171,7 +171,8 @@ public void submit(final VariantContext variantContextToOutput) {
 
     //funky logic for DRAGEN GVCFs where call may not match PLs
     private boolean isHomRef(final Genotype g) {
-        return (g.isHomRef() && !g.hasPL()) || (g.hasPL() && g.getPL()[0] == 0);
+        //consider ./. with no PL a GQ0 hom ref, as if for [0,0,0] PLs
```

Comment:
Sounds like a very long stretch too see a hom-ref something that has no information whatsoever. I guess you know what you are doing.

Reply:
Or perhaps you should name this method different if encompasses more than hom-refs